### PR TITLE
feature/encryption_integration

### PR DESCRIPTION
**Pull request by marpuch**
Small stuff with basic decisions how to integrate the encryption service into the environment service.

### CODE REVIEW

**Comment by przwojtkow** [comment]
Resolved small conflict. Develop merged.

File: allure-framework-modules/allure-core-module/src/test/java/com/capgemini/ntc/test/core/base/environment/EnvironmentMainTest.java [hunk 1]
```diff
@@ -117,6 +119,33 @@ public void envLogTest() {
...
+	public void testNoDataEncryptionServicePresent() {
+		// given
+		systemUnderTest.setEnvironment("DEV");
+		systemUnderTest.setDataEncryptionService(null);
+		
+		// when
+		String value = systemUnderTest.getValue("PASSWORD");
+		
+		// then
+		assertEquals("ENC(gD6S9sHAhNb6kVsCsZd81A==)", value);
```
Comment (przwojtkow) [inline allure-framework-modules/allure-core-module/src/test/java/com/capgemini/ntc/test/core/base/environment/EnvironmentMainTest.java]: All expected values in assertions might be assign to variable before usage.

**Comment by przwojtkow** [comment]
Most changes marked as a formatter differences. Formatter file can be found at path: devonfw-testing\resources\style_format

**Comment by marpuch** [comment]
I did and still do have the formatter set properly. Funny thing this happened. Duno why. Will look closely if it happens again in the future.

I believe I did make the requested changes. Please look again.

**Comment by marpuch** [comment]
Is there something else that needs to be done, or can the merge actually be performed?

**Comment by robert-duda-cap** [comment]
For me it looks well. The only think I could have doubt about is method optionalDecrypt with two return points. If it is allowed in project, well done and go ahead.

**Comment by robert-duda-cap** [comment]
I am sorry, I thought that I am closing comment box.

**Comment by lstefaniszyn** [comment]
@przwojtkow :  is it ok to merge ?

**Pull request merged**